sass/themes/_error.scss

```
@@ -723,11 +761,14 @@ category: Errors
 * <https://references.modernisation.gouv.fr/rgaa/criteres.html#crit-8-4>
 * <https://www.w3.org/TR/WCAG20/#meaning-doc-lang-id>
 * <https://www.w3.org/TR/WCAG-TECHS/H57.html>
+* <https://html.spec.whatwg.org/#attr-lang>
+* <https://tools.ietf.org/html/bcp47#section-2.1.1>
+* <https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/lang>
```

A few links to add, let me do this:
```suggestion
* <https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/lang>
* <https://www.w3.org/WAI/WCAG21/Understanding/language-of-page>
* <https://www.w3.org/WAI/WCAG21/Understanding/language-of-parts.html>
```